src/components/common/autocomplete-input.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FunctionComponent } from "react";
+import Autosuggest from "react-autosuggest";
+import styled from "@emotion/styled";
+
+const AutoCompleteStyle = styled.div`
```

Comment:
please hide the horizontal scrollbar (I mean like really shouldn't see anything)
 same for vertical, it should be hidden until the container is too big and need to overflow

Reply:
mm, I don't see if appearing if it doesn't overflow. However, just in case, I will update it to overflow:auto